dashboard/client/src/App.tsx

```diff
@@ -165,39 +166,46 @@ const App = () => {
                       }
                       path=""
                     />
-                    <Route element={<NodeDetailPage />} path="nodes/:id" />
                   </Route>
+                  <Route element={<NodeDetailPage />} path="nodes/:id" />
                 </Route>
                 <Route element={<JobsLayout />} path="jobs">
                   <Route element={<JobList />} path="" />
-                  <Route element={<JobDetailLayout />} path=":id">
-                    <Route
-                      element={
-                        <SideTabPage tabId="info">
-                          <JobDetailInfoPage />
-                        </SideTabPage>
-                      }
-                      path="info"
-                    />
+                  <Route element={<JobPage />} path=":id">
+                    <Route element={<JobDetailLayout />} path="">
```

Comment:
nit: what's the convention for calling it `Page` and `Layout`? ie. if there's anything enforced where `Layout` is a descendant of `Page`, etc.

Reply:
Layout generally means no content, just tabs or somethign like that.
Page has user content. 

It's not really a bulletproof convention